### ISSUE TITLE
this software needs to improvement more

### ISSUE THREAD

**Issue by Spiral2839** [YTP-Plus/YTPPlusStudio#1]
I was thinking of fixing/improve stuff, like I wonder...

- [ ] Cursor Design
- [ ] Add Buttons
- [ ] New Effects [Like Datamosh, Ricecake, and etc.]
- [ ] Added VideoEditBot section
- [ ] Editor Section
- [ ] Splash Screen
- [ ] Added Render Canceled
- [ ] New Render Completed/Failed Sound
- [ ] New Preview Section [added Play, Pause, Timeline, Next/Previous Frame]

I hope you added some more stuff soon!

**Comment by nuppington-bit** [comment]
bump

**Comment by nuppington-bit** [comment]
> * [ ]  Cursor Design
> * [ ]  Splash Screen

Both of these exist already, could you clarify what you'd want it to look like instead of what we have now?

**Comment by TeamPopplio** [comment]
## I'm a bit confused here but let me address some of these anyway:

>  Cursor Design

It's programmer art, everything is and is due to be replaced sometime in the future. I may also make an option to disable the cursor.

> Add Buttons

This is too general and you're likely referring to the buttons that @nuppington-bit implemented subsequently after this issue. There are still missing buttons (generation, plugins, options, etc) but that's because this software is still in development stages.

> New Effects [Like Datamosh, Ricecake, and etc.]

I have no idea what these examples are but yes, there will be new effects and it won't just be limited to ffmpeg either. The plugin system would allow for practically any code that can input and output video files to be a plugin.

> Added VideoEditBot section

I still don't know what this is but the true nature of YTP+ itself is to let the program do it for you, meaning you shouldn't have to edit the videos yourself.

> Editor Section

See above.

> Splash Screen

I agree with this although LÖVE has limited support for videos in itself. This is something that could be added in the future but isn't something I want to add at the moment.

> Added Render Canceled

Since there will be a separate CLI from the UI this time around, cancelling a render would be as simple as just closing the console or CLI window when rendering starts. At least, that's how I plan for it to be implemented.

>  New Render Completed/Failed Sound

The old ones were straight from Windows 8 so this is definitely going to happen to avoid any future infringement. Sounds from https://freesound.org are likely to come.

>  New Preview Section [added Play, Pause, Timeline, Next/Previous Frame]

There won't be a preview section, you'll be given the option to open the video in your preferred video player either manually or automatically after a completed render. This is due to the amount of issues YTP++ had when it came to VLC integration.

## In short:

It's too early to make issue requests for these things. Especially since some of these are features that have been discussed to be implemented in the future. This software does not need any more improvement because it will be improved. I appreciate the issue nonetheless. Closed.
